testing/testing_page.html

```diff
@@ -70,55 +70,175 @@ <h1>Widgetastic About Modal</h1>
 
 
 <!--------------------------------- Bootstrap-Datepicker------------------------------------------>
-<div class="datepicker_container" align="center" style="width: 300px; text-align:center">
-  <div class="datepicker_readonly">
-    <label class="control-label " for="date_readonly">
-      Bootstrap datepicker readonly
-    </label>
-    <input class="form-control" id="date_readonly" name="date_readonly" readonly="readonly" autoclose="true" placeholder="MM/DD/YYYY" data-date-format="mm/dd/yyyy" type="text"/>
-  </div>
-     
-  <div class="datepicker_readwrite">
-    <label class="control-label " for="date_readwrite">
-      Bootstrap datepicker readwrite
-    </label>
-    <input class="form-control" id="date_readwrite" name="date_readwrite" autoclose="true" placeholder="MM/DD/YYYY" data-date-format="mm/dd/yyyy" type="text"/>
-  </div>
+  <div class="datepicker_container" align="center" style="width: 300px; text-align:center">
```

Comment:
This should all just be indentation changes, to get this in-line with the body element. I am not intending to change the datepicker widget on the test page.